gcsweb/Makefile

```diff
@@ -54,7 +54,7 @@ endif
 
 IMAGE := $(REGISTRY)/$(BIN)-$(ARCH)
 
-BUILD_IMAGE ?= golang:1.9-alpine
+BUILD_IMAGE ?= golang:1.9.3-alpine
```

Comment:
Do we want to put in the minor version here? `golang:1.9-alpine` will map to the latest 1.9